run_splunk_container.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+IMAGE_NAME="custom-splunk"
+
+get_random_port() {
+  echo $(( 49152 + RANDOM % (65535 - 49152 + 1) ))
+}
+
+API_PORT=$(get_random_port)
+WEB_PORT=$(get_random_port)
```

Comment:
we may want to make sure we don't get the same port (although a very slim chance)